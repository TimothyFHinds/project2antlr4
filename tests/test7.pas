program test7;

var 
a, b, ret: REAL;

//user defined functions
//https://www.tutorialspoint.com/pascal/pascal_functions.htm
function max(num1, num2: real): real;
var 
    //local variable declaration
    result: real;

begin
    if(num1 > num2) then
    result := num1

    else
        result := num2;
    
    max := result;
end;

//main block
begin
    a := 100;
    b := 200;
    //function call to find max value
    ret := max(a, b);
    writeln('The values are: ');
    writeln(a);
    writeln(b);

    writeln('Max value is ');
    writeln(ret);
end.