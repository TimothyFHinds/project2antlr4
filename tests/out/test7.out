'The values are: '
100.0
200.0
'Max value is '
200.0
